iset.mm

```diff
@@ -135016,6 +135038,642 @@ the base set to the (extended) reals and which is nonnegative, symmetric
+      setsmsbasg.d $e |- ( ph -> ( MetOpen ` D ) e. W ) $.
+      $( The base set of a constructed metric space.  (Contributed by Mario
+         Carneiro, 28-Aug-2015.) $)
+      setsmsbasg $p |- ( ph -> X = ( Base ` K ) ) $=
+        ( cbs cfv cnx wcel wne c1 c9 tsetndx cts cmopn cop csts co baseslid 1re
+        wceq 1lt9 ltneii basendx neeq12i mpbir cn 9nn eqeltri setsslnid syl2anc
+        fveq2d 3eqtr4d ) ADMNZDOUANZBUBNZUCUDUEZMNZGCMNADEPVCFPVAVEUHKLEVCVBMFD
+        UFOMNZVBQRSQRSUGUIUJVFRVBSUKTULUMVBSUNTUOUPUQURHACVDMJUSUT $.
+
+      $( The distance function of a constructed metric space.  (Contributed by
+         Mario Carneiro, 28-Aug-2015.) $)
+      setsmsdsg $p |- ( ph -> ( dist ` M ) = ( dist ` K ) ) $=
+        ( cds cfv cnx cts wcel wceq wne c9 cmopn cop csts co dsslid cdc 9re 1nn
+        c1 c2 2nn0 9nn0 9lt10 declti dsndx tsetndx neeq12i mpbir cslot tsetslid
+        gtneii cn simpri setsslnid syl2anc fveq2d eqtr4d ) ADMNZDOPNZBUANZUBUCU
+        DZMNZCMNADEQVJFQVHVLRKLEVJVIMFDUEOMNZVISUIUJUFZTSTVNUGUIUJTUHUKULUMUNVA
+        VMVNVITUOUPUQURPVIUSRVIVBQUTVCVDVEACVKMJVFVG $.
+
+      $( The topology of a constructed metric space.  (Contributed by Mario
+         Carneiro, 28-Aug-2015.)  (Revised by Jim Kingdon, 7-May-2023.) $)
+      setsmstsetg $p |- ( ph -> ( MetOpen ` D ) = ( TopSet ` K ) ) $=
+        ( cmopn cfv cnx cts cop csts co wcel tsetslid setsslid syl2anc fveq2d
+        wceq eqtr4d ) ABMNZDOPNUGQRSZPNZCPNADETUGFTUGUIUEKLEUGPFDUAUBUCACUHPJUD
+        UF $.
+    $}
+  $}
+
+  ${
+    $d x y A $.  $d r w x y z D $.  $d r x y J $.  $d x y z R $.  $d w x y S $.
+    $d r y N $.  $d r w x y z P $.  $d z T $.  $d r w x y z X $.
+    mopni.1 $e |- J = ( MetOpen ` D ) $.
+    $( An open set of a metric space includes a ball around each of its points.
+       (Contributed by NM, 3-Sep-2006.)  (Revised by Mario Carneiro,
+       12-Nov-2013.) $)
+    mopni $p |- ( ( D e. ( *Met ` X ) /\ A e. J /\ P e. A ) ->
+                 E. x e. ran ( ball ` D ) ( P e. x /\ x C_ A ) ) $=
+      ( vy cxmet cfv wcel cv wss wa cbl crn wrex wral wi elmopn simplbda anbi1d
+      wceq eleq1 rexbidv rspccv syl 3impia ) CFIJKZBEKZDBKZDALZKZULBMZNZACOJPZQ
+      ZUIUJNHLZULKZUNNZAUPQZHBRZUKUQSUIUJBFMVBHABCEFGTUAVAUQHDBURDUCZUTUOAUPVCU
+      SUMUNURDULUDUBUEUFUGUH $.
+
+    $( An open set of a metric space includes a ball around each of its points.
+       (Contributed by NM, 2-May-2007.)  (Revised by Mario Carneiro,
+       12-Nov-2013.) $)
+    mopni2 $p |- ( ( D e. ( *Met ` X ) /\ A e. J /\ P e. A ) ->
+                 E. x e. RR+ ( P ( ball ` D ) x ) C_ A ) $=
+      ( vy cxmet cfv wcel w3a cv wss wa cbl crn wrex co crp mopni mopnss sselda
+      wb blssex adantlr syldan 3impa mpbid ) CFIJKZBEKZDBKZLDHMZKUMBNOHCPJZQRZD
+      AMUNSBNATRZHBCDEFGUAUJUKULUOUPUDZUJUKOZULDFKZUQURBFDBCEFGUBUCUJUSUQUKHBCD
+      FAUEUFUGUHUI $.
+
+    $( An open set of a metric space includes an arbitrarily small ball around
```

Comment:
What is a "small ball", i.e., when is a ball "small"?

Reply:
It is small when the radius (`x`) is less than `R` which is arbitrary (that is, the theorem holds for any value of `R` in `RR+`).

Perhaps the wording "An open set of a metric space includes a ball around each of its points which can be made as small as desired" is better?

But I'm not sure I like that better than what is there now.

Did you have any wording in mind?

Reply:
Ah OK, I missed the "arbitrarily"... I think "arbitrarily small" is good enough.